sycl/source/detail/scheduler/commands.cpp

```diff
@@ -1814,9 +1820,14 @@ cl_int ExecCGCommand::enqueueImp() {
     // piEnqueueNativeKernel takes arguments blob which is passes to user
     // function.
     // Reserve extra space for the pointer to CGExecKernel to restore context.
-    std::vector<void *> ArgsBlob(HostTask->MArgs.size() + 1);
+    std::vector<void *> ArgsBlob(HostTask->MArgs.size() + 2);
     ArgsBlob[0] = (void *)HostTask;
-    void **NextArg = ArgsBlob.data() + 1;
+    {
```

Comment:
In the future would we want this Blob[1] position to be used as a bitfield?   If that seems like a good idea, maybe we should leave a comment for the future.  

Though, I can't think of many uses, except adding support for variable length arg lists.